pkg/node/peer_manager/peer_manager.go

```diff
@@ -109,27 +109,28 @@ func (a *PeerManagerImpl) NewConnection(p peer.Peer) error {
 
 	if p.Handshake().Version.CmpMinor(a.version) >= 2 {
 		err := errors.Errorf(
-			"versions are too different, current %s, connected %s",
+			"versions are too different, current %s, connected %s (peer '%s')",
 			a.version.String(),
 			p.Handshake().Version.String(),
+			p.ID(),
 		)
 		a.restrict(p, now, err.Error())
 		_ = p.Close()
-		return proto.NewInfoMsg(err)
+		return err
 	}
 	if p.Handshake().AppName != a.networkName {
 		err := errors.Errorf("peer '%s' has the invalid network name '%s', required '%s'",
 			p.ID(), p.Handshake().AppName, a.networkName)
 		a.restrict(p, now, err.Error())
 		_ = p.Close()
-		return proto.NewInfoMsg(err)
```

Comment:
And this one. Those errors are very common and they will bloat the log with error messages. Has meaning only on DEBUG level.

Reply:
The same as above, fixed.